README.md

```diff
@@ -17,7 +17,7 @@ Every argument is optional.
 | Input              | Description                                                                                                                       | Default |
 |--------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------|
 | `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages in the workspace are processed. | |
```

Comment:
```suggestion
| `package`            | The package whose API to check for semver (in Package Id Specification format, see https://doc.rust-lang.org/cargo/reference/pkgid-spec.html for reference). If not set, all packages defined in the Cargo.toml file are processed. | |
```
Tweak because if the `manifest-path` points to a crate manifest within a workspace, then only that crate will be checked even if `package` isn't set.

Reply:
You're right, I missed this case